### PR TITLE
Adding CVE's Exploitation attempt detection: Year - 2010

### PR DESCRIPTION
**Pull request by pbssubhash**
I've taken reference from Nuclei Templates - https://github.com/projectdiscovery/nuclei-templates. Nuclei is a tool used for automated scanning of web vulnerabilities. I built an automated tool to convert a Nuclei template to Sigma rule. Using this, we can identify exploitation attempts over external attack surface.

Team, please review my PR. 🙏

### CODE REVIEW

**Comment by Neo23x0** [comment]
Please use title casing for the title field - e.g. `Suspicious PowerShell CommandLine Params` instead of `Suspicious powershell commandline params`

https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#title

File: rules/web/web_cve_CVE-2010-0943_exploitation_attempt.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+title: CVE-2010-0943 exploitation attempt
+id: fa7e28e3-d286-4c38-b6df-41eb004b66ba
+author: Subhash Popuri (@pbssubhash)
+date: 25/08/2021
```
Comment (Neo23x0) [inline rules/web/web_cve_CVE-2010-0943_exploitation_attempt.yaml:4]: Please use a different timestamp format, e.g.:
```
date: 2021/08/23
```

File: rules/web/web_cve_CVE-2010-0943_exploitation_attempt.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+status: experimental
+description: Directory traversal vulnerability in the JA Showcase (com_jashowcase)
+  component for Joomla! allows remote attackers to read arbitrary files via a .. (dot
+  dot) in the controller parameter in a jashowcase action to index.php.
+references:
+- https://www.exploit-db.com/exploits/11090
```
Comment (Neo23x0) [inline rules/web/web_cve_CVE-2010-0943_exploitation_attempt.yaml:10]: Please indent these lines

**Comment by Neo23x0** [comment]
Sorry, I have more requests 😅

- Please use only lower case characters in the file names (`CVE` > `cve`)
- Please add a product and component name in the title, so that a user doesn't have to lookup the CVE, e.g. `CVE-2010-0943 exploitation attempt` to `CVE-2010-0943 Joomla JaShowcase Exploitation Attempt`.

**Comment by pbssubhash** [comment]
Hey,

Actually it's an automated script that's converting nuclei templates to sigma rules. Let me try to incorporate the changes.. 

Thanks for the feedback.

**Comment by pbssubhash** [comment]
@Neo23x0 , How can I debug this error? Please help me.

image

**Comment by frack113** [comment]
HI,
You have change in **sysmon_detect_powerup_dllhijacking.yml** `category: file_event` to `service: sysmon`.
So now sigmac expect a Eventid.
You have just to **revert** sysmon_detect_powerup_dllhijacking.yml to the original.

**Comment by frack113** [comment]
My bad I have miss that
```bash
======================================================================
ERROR: test_invalid_logsource_attributes (__main__.TestRules)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/test_rules.py", line 589, in test_invalid_logsource_attributes
    for key in logsource:
TypeError: 'NoneType' object is not iterable
```
Your rules must have a logsource [wiki](https://github.com/SigmaHQ/sigma/wiki/Taxonomy)
```yaml
logsource:
   category: webserver
```

**Comment by pbssubhash** [comment]
@frack113, I've made the required changes. 
Only one thing which I couldn't do it, as suggested by @Neo23x0 is regarding the CVE title. This is an automated converter and the required information isn't available in Nuclei templates. However, information regarding the CVE is there in description. Please cut me some slack and accept the PR. 1K+ similar rules coming your way if you accept the PR..

**Review by frack113 — APPROVED** [review]

**Comment by Neo23x0** [comment]
Okay, I see. The automatic converter cannot do that for us.

**Pull request merged**